be/src/vec/exec/scan/vfile_scanner.cpp

```diff
@@ -1110,4 +1110,11 @@ Status VFileScanner::close(RuntimeState* state) {
     return Status::OK();
 }
 
+void VFileScanner::try_stop() {
+    VScanner::try_stop();
+    if (_io_ctx) {
+        _io_ctx->should_stop = true;
```

Comment:
Need `should_stop` be atomic?

Reply:
`should_stop` only needs the final consistency.